models/spring-ai-bedrock-converse/src/test/java/org/springframework/ai/bedrock/converse/BedrockConverseChatClientIT.java

```diff
@@ -70,7 +70,9 @@ void call() {
 				.system(s -> s.text(this.systemTextResource)
 						.param("name", "Bob")
 						.param("voice", "pirate"))
-				.user("Tell me about 3 famous pirates from the Golden Age of Piracy and what they did")
+				.user(u -> u.text("Tell me about 3 famous pirates from the Golden Age of Piracy and what they did")
+						.param("requestId", "1234")
```

Comment:
should be `.metadata` to verify the request metadata.

Reply:
It would be nice to verify the ConverseRequest updated with metadata. Also, Is there a way to verify the response based on the request metadata being sent?